listen.py

```diff
@@ -43,6 +43,41 @@ def from_json(cls, j):
             data=j.get('track_metadata')
         )
 
+    @classmethod
+    def from_influx(cls, row):
+        """ Factory to make Listen objects from an influx row
+        """
+        dt = datetime.strptime(row['time'] , "%Y-%m-%dT%H:%M:%SZ")
```

Comment:
This is the only place that you use dictionary access instead of `.get`. Do we have tests to verify the minimum data which we get from influx?  If not, can you open a ticket to make sure that we write some?